src/main/java/nl/martijndwars/webpush/ClosableCallback.java

```diff
@@ -0,0 +1,45 @@
+package nl.martijndwars.webpush;
```

Comment:
I'm noticing that there's a maven artifact for `nl.martijndwars.webpush`. Can we use that, instead of forking the code?

```
<dependency>
    <groupId>nl.martijndwars</groupId>
    <artifactId>web-push</artifactId>
    <version>5.1.1-SNAPSHOT</version>
</dependency>
```